kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStateList.js

```diff
@@ -92,6 +114,15 @@ define([
          * @param {Object} message
          */
         function handleJobStatusUpdate(message) {
+            if (message.jobState.batch_size && message.jobState.child_jobs.length === 0) {
```

Comment:
This works for the current message data structure that comes back from kbParallel, but we should check with Boris what the ee2 will return once all those changes are in place.